src/virtual_machine/lc3_virtual_machine.rs

```diff
@@ -188,9 +188,19 @@ impl LC3VirtualMachine {
         self.decode_instruction(instruction);
     }
 
-    fn sign_extend(mut value_to_extend: u16, ammount_of_bits: u16) -> u16 {
-        if (value_to_extend >> (ammount_of_bits - 1) & 0b1) == 1 {
-            value_to_extend |= 0xFFFF << ammount_of_bits;
+    pub fn next_instructions(&mut self, amount_of_instructions: u16) {
```

Comment:
Maybe steps is a brief but clear name:
```suggestion
    pub fn next_instructions(&mut self, steps: u16) {
```